modules/sdk/project-templates/project-templates-theme/src/main/resources/archetype-resources/build.gradle

```diff
@@ -14,11 +14,11 @@ apply plugin: "com.liferay.portal.tools.theme.builder"
 
 dependencies {
 #if (${liferayVersion.startsWith("7.0")})
-	cssBuilder group: "com.liferay", name: "com.liferay.css.builder", version: "3.0.0"
+	cssBuilder group: "com.liferay", name: "com.liferay.css.builder", version: "3.1.4"
 #elseif (${liferayVersion.startsWith("7.1")} || ${liferayVersion.startsWith("7.2")} || ${liferayVersion.startsWith("7.3")})
-	cssBuilder group: "com.liferay", name: "com.liferay.css.builder", version: "3.0.3"
+	cssBuilder group: "com.liferay", name: "com.liferay.css.builder", version: "3.1.4"
 #else
-    cssBuilder group: "com.liferay", name: "com.liferay.css.builder", version: "3.1.3"
+    cssBuilder group: "com.liferay", name: "com.liferay.css.builder", version: "3.1.4"
```

Comment:
Since these are now all identical, we can completely remove this conditional.

Reply:
> Since these are now all identical, we can completely remove this conditional.

Hi @drewbrokke , this changes are auto generated after executing "gw updateFielVersions", I think it's not expected to modify the css-builder version for all liferay version templates, since there will be some build issues when using lastest css-builder version for 7.0, 7.1, 7.2 and 7.3. Maybe I should change the version manually in the templates?

Reply:
yeah, I think that's probably better. If older versions of the plugin may be required for older versions of Liferay, we should change these manually.